src/ServiceControl.Monitoring/Hosting/Commands/RunCommand.cs

```diff
@@ -13,8 +13,8 @@ public override async Task Execute(Settings settings)
 
             var hostBuilder = WebApplication.CreateBuilder();
             hostBuilder.AddServiceControlMonitoring((_, __) => Task.CompletedTask, settings, endpointConfiguration);
-            using var app = hostBuilder.Build();
 
+            var app = hostBuilder.Build();
```

Comment:
This is a general comment. Not something we have to address though. I was wondering if it would still be better from a code flow perspective to do using. The app is disposable and from that perspective it would make sense to add a using. Otherwise we are relying on knowledge of implementation details of `RunAsync` which I'm a little bit on the fence but I can see the argument of "yeah but how the code is right now is more aligned with the default templates of MS that also relies on RunAsync doing the dispose"